x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -99,15 +100,15 @@ export const AnalyticsCollectionIntegrate: React.FC<AnalyticsCollectionIntegrate
 
       <EuiSpacer size="l" />
       <EuiText size="s">
-        <p>
+        <EuiMarkdownFormat textSize="relative">
```

Comment:
suggestion: EuiMarkdownFormat is really meant to display markdown text, not to add HTML formatting to text we're using. A more idiomatic way to accomplish what you're trying to do is with FormattedMessage, like so:
https://github.com/elastic/kibana/blob/c9eebf262c65d6c3104592b6ea0bb61f1d6e0d1f/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/crawl_custom_settings_flyout/crawl_custom_settings_flyout_seed_urls_panel.tsx#L119

I'd also suggest renaming the PR: it's not really about adding Markdown support, it's just about adding some text formatting to a single word.

Reply:
Wow, huge thanks!